constraint/pkg/client/drivers/local/local.go

```diff
@@ -492,3 +497,192 @@ func (d *driver) Dump(ctx context.Context) (string, error) {
 
 	return string(b), nil
 }
+
+func MapModules(templ *templates.ConstraintTemplate, extern []string) (string, []string, error) {
+	if err := validateTargets(templ); err != nil {
+		return "", nil, err
+	}
+	targetSpec := templ.Spec.Targets[0]
+	targetHandler := targetSpec.Target
+	kind := templ.Spec.CRD.Spec.Names.Kind
+	libPrefix := templateLibPrefix(targetHandler, kind)
+
+	rr, err := regorewriter.New(
+		regorewriter.NewPackagePrefixer(libPrefix),
+		[]string{"data.lib"},
```

Comment:
Use a string constant for "data.lib" right now it's just a magic string. A good name for this would be "libRoot" with a comment saying that it is the root of all libraries a ConstraintTemplate declares.